### PR TITLE
Update user guide.md

### PR DESCRIPTION
**Pull request by Mamtha3005**

### CODE REVIEW

**Comment by Mamtha3005** [comment]
Ready to Review @CS2103AUG2016-W15-C3/documentation

File: docs/UserGuide.md [hunk 1]
```diff
@@ -9,21 +9,21 @@
...
-   e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window. 
+    > ![gui listToday](gif/listToday.gif)<br>
+   e.g. typing **`list today`** and pressing <kbd>Enter</kbd> will list the tasks of today.
 5. Some example commands you can try:
    * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` : 
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
-   * **`exit`** : exits the app
+   * **`add`** `add buy MA1101R textbook today` : adds a task called �buy MA1101R textbook� to be done by today
```
Comment (marcusngwenjian) [inline docs/UserGuide.md]: An extra 'add' was written

File: docs/UserGuide.md [hunk 2]
```diff
@@ -41,38 +41,38 @@ Format: `help`
...
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+> * The order of the keywords does not matter. e.g. `chicken egg` will match `egg chicken`
+> * Only the task item is searched, not the tags.
+> * Only full words will be matched e.g. `chicken` will not match `chickens`
+> * Taskss matching at least one keyword will be returned (i.e. `OR` search).
```
Comment (marcusngwenjian) [inline docs/UserGuide.md]: Typo 'Tasks'

File: docs/UserGuide.md [hunk 3]
```diff
@@ -119,16 +112,17 @@ There is no need to save manually.
```
Comment (marcusngwenjian) [inline docs/UserGuide.md:112]: Under Saving the data, "Address Book" should be changed to "Taskell"

File: docs/UserGuide.md [hunk 4]
```diff
@@ -119,16 +112,17 @@ There is no need to save manually.
 
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with
```
Comment (marcusngwenjian) [inline docs/UserGuide.md]: Replace "app" with "application"

**Comment by marcusngwenjian** [comment]
@Mamtha3005 Reviewed. Please make the respective changes.

File: docs/UserGuide.md (hunk 1 above)
Comment (Mamtha3005) [reply docs/UserGuide.md]: That is not 2 add... 1 is saying what command and the other is an example of how to use the command,

**Comment by Mamtha3005** [comment]
@CS2103AUG2016-W15-C3/documentation Ready for review

**Comment by marcusngwenjian** [comment]
Reviewed.
@CS2103AUG2016-W15-C3/integration Ready for integration

**Pull request merged**